crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -176,7 +176,9 @@ impl AdminInterfaceApi for RealAdminInterfaceApi {
             ListCellIds => {
                 let cell_ids = self
                     .conductor_handle
-                    .list_cell_ids(Some(CellStatus::Joined));
+                    .running_cell_ids(Some(CellStatus::Joined))
```

Comment:
This seems incorrect. Wouldn't an admin client want to know all the cells?

If this *is* correct, it may need a comment so we don't think it's wrong in the future.

Reply:
I updated the doc for this admin method